data/common.yaml

```diff
@@ -0,0 +1,4 @@
+---
+sftp_jail::password_authentication:
+  sftp_admins: false
+  sftp_users: false
```

Comment:
OK, how would you name the variable? IIRC I did put both in one variable, because the variable names would be very long: $password_authentication_admins is a pretty long name.

Reply:
A long name is ok with me. You can shorten authentication to auth if you want. An alternative is `Enum['all', 'admins', 'users', 'none'] $password_authentication`. I'm not sure which one of them is best so I'll leave it up to you to consider.

Reply:
fixed: https://github.com/voxpupuli/puppet-sftp_jail/pull/69/commits/d1b80a196eaa6a70286f3aee05fcfdf562e4f87d